index.js

```diff
@@ -4,7 +4,7 @@
 var path = require('path');
 var arrify = require('arrify');
 var assign = require('object-assign');
-var formatter = require('stylelint/dist/formatters/stringFormatter').default;
+var formatter = require('stylelint/lib/formatters/stringFormatter');
```

Comment:
I recommend you use the public API here rather than an undocumented path:

```js
var formatter = require('stylelint').formatters.string;
```

It'll save you from having to make changes like this in the future.